server/integration/endpoint/src/test/resources/org/infinispan/server/endpoint/endpoint-9.0.xml

```diff
@@ -18,6 +18,12 @@
         </sasl>
       </authentication>
    </hotrod-connector>
+    <hotrod-connector name="hotrod5" socket-binding="hotrod" cache-container="default">
```

Comment:
I've been playing with this for a while and I think the SNI part should be moved under `authentication`.